### ISSUE TITLE
Transition: There is no css animation matching the one you specified. scale out 

### ISSUE THREAD

**Issue by wuxianliang** [richsilv/semantic-ui-modal#2]
Dear @richsilv 
First of all, Thank you very very much! I am using this package in my personal site which is still under developing. 

When I close the modal, there is an error that I can't see the closing transition. 

Thank you!

**Comment by adammhaile** [comment]
Is there anyway to suppress this error? I understand that it seems to be expected, but it makes me twitch a bit every time I see the error appear.

**Comment by startswithaj** [comment]
Do you guys understand what's causing this?

**Comment by joetm** [comment]
Having the same issue.
Another person mentioned that the specific animation was not present in jquery:
http://stackoverflow.com/a/27807854/426266

**Comment by adrazich** [comment]
I'm getting this error when the page just loads and you mouse over something with a popup very quickly. Like moving the mouse over it to get to something else. It stops shouting about it after a few seconds doing it repeatedly.

**Comment by adammhaile** [comment]
That is *exactly* what I am seeing as well.

**Comment by startswithaj** [comment]
I don't think it has anything to do with the animation not existing. The error is a misnomer. If you put a breakpoint where it errors you can see that it's reusing that message in the wrong spot. It can't find the animation because the element doesn't exist at the time it tries to scale it out. There is some race condition here and that's why it doesn't happen all the time.  @adrazich points out a good way to repro. Yet to track down exactly what is causing it. Happens mostly with the popup element

**Comment by startswithaj** [comment]
Another couple of things to clear up is that it's not looking for the animation in jquery. It's looking for the animation defined in the style sheet.

**Comment by joetm** [comment]
I get this error when transitioning between two modals.
I assume that loading the correct semantic UI CSS module could solve the problem.
Just another unspecified dependency in this framework.

What can I add to my site's CSS to prevent this error?
Maybe there is a 'dummy' animation that I could add?

**Comment by joetm** [comment]
I did check and found out that I am loading the semantic UI scaleOut transition definitions (both the css and js).

**Comment by richsilv** [comment]
I've just published a new version which works fine with the latest version of the templating module (1.1.1) in the [demo](https://github.com/richsilv/meteor-semantic-ui-modal-demo).  I'll admit that I don't know what it is that's changed, but please give it a go and let me know how it works for your use cases (`meteor update` should be all you need).
